tests/test_restaurant_item.py

```diff
@@ -0,0 +1,26 @@
+def test_get_nonexisten(test_client, get_mocker):
+    response = test_client.get('/api/restaurants/3000')
+    assert response.status_code == 404
+    assert {"description": "Restaurant with such ID doesn\'t exist", "name": "Not Found"} == response.get_json()
+
+
+def test_put_nonexisten(test_client, get_mocker):
+    response = test_client.put('/api/restaurants/3000', json={"name": "Hotfix"})
+    assert response.status_code == 404
+    assert {"description": "No restaurant to update. Restaurant with such ID doesn\'t exist",
+            "name": "Not Found"} == response.get_json()
+
+
+def test_get_one(test_client, get_mocker):
+    response = test_client.get('/api/restaurants/0')
+    data = {"address": "Minsk", "id": 0, "name": "Vasilki", "phone_number": "+375297777777",
```

Comment:
if `restaurants` mock is used, then it is possible to avoid copy-paste here to like: `data = restaurants[0]`

Reply:
fixed